particula/__init__.py

```diff
@@ -19,4 +19,11 @@
 # u is the unit registry name.
 u = UnitRegistry()
 
+# Temperature has the quirky issue that mathematical operations on temperature
+# are problematic. e.g. 298.15K + 273.15K != 25C + 0C.
+# The following setting converts all offset units to base units prior to
+# any operations.
+# https://pint.readthedocs.io/en/latest/nonmult.html
+u.autoconvert_offset_to_baseunit = True
```

Comment:
Can you explain how this relates to `u.K` vis-a-vis `u.degK`?

Reply:
Or is `u.K` simply incorrect?

Reply:
That's a good question, and I'm investigating further. Pint defines kelvin as degK, I will dig further to see if they are equivalent.

Reply:
Thank you for looking in these things! I tried figuring this out and I am still a bit confused. Note `u.K` worked fine previously, so they could be interchangeable, but I am not sure.

Reply:
After testing, `u.K` and `u.degK` are equivalent, and they interact normally together. I imagine both were implemented because `u.K` is the more accurate definition, and `u.degK` is for clarity and uniformity with `u.degF` and `u.degC`. I am indifferent between the two, but we should try to stick with one. Any opinions?